ntttcp/ntttcp.Config.json

```diff
@@ -0,0 +1,61 @@
+{
+    "NtttcpDefault": {
+        "Iterations": 1,
+        "StartPort": 50000,
+        "tcp" : {
+            "BufferLen"     : [65536],
+            "Connections"   : [64],
+            "OutstandingIo" :[2, 16],
+            "Options"       : ""
+        },
+        "udp" : {
+            "BufferLen"     : [65536],
+            "Connections"   : [64],
+            "OutstandingIo" : null,
+            "Options"       : ""
+        },
+        "Warmup"   : 2,
+        "Cooldown" : 2,
+        "Runtime"  : 10
+    },
+
+    "NtttcpAzure": {
+        "Iterations" : 5,
+        "StartPort"  : 50000,
+        "tcp" : {
+            "BufferLen"     : [65536],
+            "Connections"   : [1, 64],
+            "OutstandingIo" :[2, 16],
+            "Options": ""
+        },
+        "udp" : {
+            "BufferLen"     : [1472],
+            "Connections"   : [1, 64],
+            "OutstandingIo" : null,
+            "Options"       : ""
+        },
+        "Warmup"   : 2,
+        "Cooldown" : 2,
+        "Runtime"  : 90
+    },
+
+    "NtttcpDetail": {
+        "Iterations": 3,

```

Comment:
Do we want this to be 5 as well (like latte ? or 3 for ntttcp and 5 for latte) ?

Reply:
I think I changed this to 3 based on - https://github.com/microsoft/NetPerfTest/pull/28#discussion_r616133367 

Should I change it back?